.autoprefixer.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
💡 No need for this anymore; PostCSS will read `browserlist` from the package file.